src/Homework7/Task2.java

```diff
@@ -0,0 +1,30 @@
+package Homework7;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class Task2 {
+    public static void main(String[] args) {
+        try {
+            List<String> lines = new ArrayList<>();
+            BufferedReader reader = new BufferedReader(new FileReader(".\\src\\Homework7\\poem.txt"));
+            String line;
+            while ((line = reader.readLine()) != null) {
+                lines.add(line);
+            }
+            reader.close();
+
+            Collections.sort(lines, Comparator.comparingInt(String::length));
```

Comment:
Can be replaced with `lines.sort(lines, Comparator.comparingInt(String::length));`

Reply:
Do I need to pass lines parameter here as it is already in the List?
Replaced with lines.sort(Comparator.comparingInt(String::length));

Reply:
Sorry, it was a copy-paste typo, you are absolutely right)